examples/bootstrap5/app.py

```diff
@@ -155,10 +188,26 @@ def test_nav():
     return render_template('nav.html')
 
 
+@app.route('/bootswatch', methods=['GET', 'POST'])
+def test_bootswatch():
+    form = BootswatchForm()
+    if form.validate_on_submit():
+        print('FFFFF', app.config['BOOTSTRAP_BOOTSWATCH_THEME'])
```

Comment:
The print statement needs to be removed.

Reply:
Fixed, was some debug remnant.